app/sentry.ts

```diff
@@ -13,8 +13,7 @@ if (!config.sentry.dsn.includes('dummy')) {
                 levels: ['error']
             }),
             new ApmIntegrations.Tracing(),
-        ],
-        tracesSampleRate: config.sentry.tracesSampleRate, 
+        ], 
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5958291)